### PR TITLE
Added QuoteNOption Component

### PR DESCRIPTION
**Pull request by suhasbansude**
Added QuoteNOption Component.
 We need to enhance UI and Animations.

image

### CODE REVIEW

**Comment by twishasaraiya** [comment]
@suhasbansude Thank you for PR. Please go through our [UI Designs](https://dribbble.com/shots/4488762-Guess-Quote-App-Screens) and make appropriate changes.

File: README.md [hunk 1]
```diff
@@ -111,3 +111,7 @@ We are using **Git Branching Model** for production and development purposes.You
  - `master` : This branch reflects the _PRODUCTION READY STATE_
 
 For more details regarding contributing refer [CONTRIBUTING.md](https://github.com/twishasaraiya/GuessTheQuote/blob/master/CONTRIBUTING.md)
+
```
Comment (twishasaraiya) [inline README.md]: Please make a separate for changes related to README.md. These changes are not connected to QuoteNOption Component.

**Comment by twishasaraiya** [comment]
@suhasbansude Can you please add a screenshot here??? It becomes easy for us to review. And please add the following
1. Create a new page as follows `/pages/Quiz` which will contain
      a) `Quiz.js`
      b) `index.js`
      c) `Quiz.css`
2. Add a route to the page `/RouteManager`
3. Call the `QuoteNOptionContainer` from the `Quiz` page

**Comment by suhasbansude** [comment]
capture

**Comment by twishasaraiya** [comment]
@suhasbansude As I said earlier, Please go through our [UI Designs](https://dribbble.com/shots/4488762-Guess-Quote-App-Screens). The color scheme in UI Design is different from what you have used. Also use `green-tick.svg` and `red-cross.svg` for answers and not `star.svg`. Once that is done the PR is ready to be merged.

If You have any doubts please let me know.

**Comment by twishasaraiya** [comment]
Also due to new PR merged, there are merge conflicts now, so please first merge existing `develop`  with your local copy to resolve the conflict.

**Comment by twishasaraiya** [comment]
Hey, @suhasbansude please add screenshot once you have made required changes to the UI. The PR is almost ready to be merged.

**Comment by suhasbansude** [comment]
@twishasaraiya ok 
I am currently working on UI.

**Comment by suhasbansude** [comment]
Please check New UI Related changes
capture1
capture2
capture3

**Comment by twishasaraiya** [comment]
@suhasbansude Great work 👍. Thank so you much. Some changes would be required when linked to the backend but merging it now.

**Pull request merged**